CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@
   - .NET desktop development
   - Visual Studio extension development.
   - Desktop development with C++
-- [Windows 10 SDK](https://dev.windows.com/en-US/downloads/windows-10-sdk)
+- [Windows 10 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk/)
```

Comment:
Would it be better to not link to the language specific version?
```suggestion
- [Windows 10 SDK](https://developer.microsoft.com/windows/downloads/windows-10-sdk/)
```

Reply:
Although this is not part of Microsoft's doc's site, this is the recommendation for docs: https://docs.microsoft.com/en-us/contribute/how-to-write-links#links-to-pages-that-are-not-part-of-the-technical-documentation-set

@mholo65 can you please commit this suggestion.